docs/guidance/build.md

```diff
@@ -39,6 +48,8 @@ Your choice of architecture can determine how frequently your data is updated, w
 ```
 **Resource:** [Technical case studies: OCDS implementation insights report](https://www.open-contracting.org/resources/technical-case-studies-ocds-implementation-insights/) provides insights into the technical choices made in OCDS implementations in Paraguay, Zambia, Colombia, Moldova and Argentina's Road Agency Vialidad.
 
+If you’re reusing an existing tool for collecting OCDS data, you should consider how you will collate and publish data collected using the tool. The [OCDS Helpdesk](../../support/#ocds-helpdesk) can help you to consider options for publication.
```

Comment:
This repeats the paragraph above, but it doesn't restore the context that the publish will not "create or update an IT system", so it's possible that an IT system-based publisher will misread this paragraph. The only new information this paragraph adds is to contact the helpdesk, so we can perhaps move that up and remove this paragraph.

Reply:
Done in https://github.com/open-contracting/standard/pull/1061/commits/1ad00bc849bc8c636c7dadeebe7f888a6252a824